diy-guide.md

```diff
@@ -99,7 +99,7 @@ Depending on the homogeneity of the group, you are may get a lot of repeat quest
 
 ## See Also
 
-* Mozilla Webmaker [Event Guides](https://webmaker.org/event-guides)
+* <s>Mozilla Webmaker [Event Guides]</s>
```

Comment:
Looks like this may have moved to https://teach.mozilla.org/events/resources/#event-guides.